cookbooks/aws-parallelcluster-config/templates/default/init/fetch_and_run.erb

```diff
@@ -23,38 +23,29 @@ function error_exit () {
   logger -t parallelcluster "${script} - $1"
   case $custom_script in
     OnNodeStart|OnNodeConfigured)
-      echo "$1 Please check /var/log/cfn-init.log in the head node, or check the cfn-init.log in CloudWatch logs. Please refer to https://docs.aws.amazon.com/parallelcluster/latest/ug/troubleshooting-v3.html#troubleshooting-v3-get-logs for more details on ParallelCluster logs." > /var/log/parallelcluster/bootstrap_error_msg
+      if [ -z "$2" ]; then
+        echo "$1 Please check /var/log/cfn-init.log in the head node, or check the cfn-init.log in CloudWatch logs. Please refer to https://docs.aws.amazon.com/parallelcluster/latest/ug/troubleshooting-v3.html#troubleshooting-v3-get-logs for more details on ParallelCluster logs." > /var/log/parallelcluster/bootstrap_error_msg
+      else
+        echo "$2 Please check /var/log/cfn-init.log in the head node, or check the cfn-init.log in CloudWatch logs. Please refer to https://docs.aws.amazon.com/parallelcluster/latest/ug/troubleshooting-v3.html#troubleshooting-v3-get-logs for more details on ParallelCluster logs." > /var/log/parallelcluster/bootstrap_error_msg
```

Comment:
[Nitpick] Would it read better to say "more details on _using_ ParallelCluster logs"? I'm not sure.

Reply:
Thanks for pointing this out. I am using the error message from the design doc. This was a team decision during the design stage. I think there's also comments in another PR about removing point to documentation in the message: https://github.com/aws/aws-parallelcluster-cookbook/pull/1632#discussion_r1082261330 Let's consider it outside this PR